flutter/lib/ui/settings/benchmarks_configuration_screen.dart

```diff
@@ -5,16 +5,17 @@ import 'package:provider/provider.dart';
 import 'package:mlperfbench/benchmark/state.dart';
 import 'package:mlperfbench/localizations/app_localizations.dart';
 import 'package:mlperfbench/resources/config_manager.dart';
+import 'package:mlperfbench/store.dart';
 import 'package:mlperfbench/ui/error_dialog.dart';
 
 class BenchmarksConfigurationScreen extends StatelessWidget {
-  final List<BenchmarksConfig> _benchmarksConfigurations;
+  final List<TaskConfigDescription> _benchmarksConfigurations;
```

Comment:
Please update all the `benchmarksConfigurations` string in the repo to match with the new name `TaskConfigDescription`.
BTW, I think `TaskFile` describes the class more accurate.

Reply:
Done.

I have renamed everything to `TaskConfig` / `Task configuration`.
This class is `...Description` because it contains filepath _and_ name.

I didn't change `benchmarksConfigurations.json` file name to avoid confusing users who already know about this file, and to avoid breaking it for users who already use this file.
I think in future we should add UI that would allow users to edit the list of configs without manually editing files, then filename will be irrelevant.

Reply:
This one also needs to be updated:
https://github.com/mlcommons/mobile_app_open/blob/903bd43c8d2faf5eff0e9104e00a5531f1c11336/flutter/documentation/custom-tasks.md?plain=1#L37

> I think in future we should add UI that would allow users to edit the list of configs without manually editing files, then filename will be irrelevant.

Please create an issue for it so we can prioritize it.

Reply:
>This one also needs to be updated:

Done

>Please create an issue for it so we can prioritize it.

Done
https://github.com/mlcommons/mobile_app_open/issues/383